README.md

```diff
@@ -1,181 +1,93 @@
 # nvim-lspconfig
+A collection of common configurations for Neovim's built-in [language server client](https://neovim.io/doc/user/lsp.html).
 
-Collection of common configurations for Neovim's built-in [language server client](https://neovim.io/doc/user/lsp.html).
-The configurations are supported on a best-effort basis, and rely on contributions 
-from regular users to stay up-to-date.
+This repo handles automatically launching and initializing language servers that are installed on your system.
 
-This repo handles automatically launching, initializing, and configuring language servers
-that are installed on your system.
+# LSP overview
+
+Neovim supports the Language Server Protocol (LSP), which means it acts as a client to language servers and includes a Lua framework `vim.lsp` for building enhanced LSP tools. LSP facilitates features like:
+
+- go-to-definition
+- find-references
+- hover
+- completion
+- rename
+- format 
+- refactor
+
```

Comment:
it looks exhaustive, add `...` ? wiith maybe a link to `:h lsp-method` so that one can find the exhaustive list.

Reply:
Done in r3 PR